carbon-design-system/carbon
# fix(data-table): correct padding for batch action buttons

In the Sketch kit and on the carbon website, batch action buttons in the DataTable have an 8px gap between text and button (effectively using the same spacing as a ghost button). The `carbon-components` styling currently treats the spacing as a normal primary button.

#### Changelog

**Changed**

- Use `$button-padding-ghost` for buttons in `.bx--action-list`
  - Set icon in button to `position: static`

#### Testing / Reviewing

1. View react storybook
2. Check "DataTable -> with batch actions"
3. Select a variable

#### Resources

*Screenshot from https://www.carbondesignsystem.com/components/data-table/usage#multi-select-batch-action*
![image](https://user-images.githubusercontent.com/28265588/75045611-dfe67600-54c3-11ea-9261-f50d3d880c2c.png)

*Screenshot from 7.9.3 react storybook*
![image](https://user-images.githubusercontent.com/28265588/75045648-f5f43680-54c3-11ea-965e-c46eef6f0bc6.png)
